doc/ArchInstall/arch-chroot_install.sh

```diff
@@ -31,6 +31,12 @@ cat << EOF > /etc/hosts
 127.0.1.1       ${HOSTNAME}
 EOF
 
+# Lock the pacman date
+# ↓↓↓ UPDATE MIRRORS HERE ↓↓↓
+cat > /etc/pacman.d/mirrorlist << EOF
+Server=https://archive.archlinux.org/repos/2021/08/29/\$repo/os/\$arch
+EOF
```

Comment:
An alternative (and I am open to opinions here) is to create a `docker/etc/pacman.d/mirrorlist` file that contains the appropriate mirror information and then download that file from the repo. Having `UPDATE_MIRRORS_HERE` everywhere that the mirrors needs to be changed is asking for someone to miss one of them.